lib/src/network/mongo_message_transformer.dart

```diff
@@ -22,6 +22,6 @@ class MongoMessageHandler {
     sink.close();
   }
   
-  StreamTransformer get transformer => new StreamTransformer.fromHandlers(
+  StreamTransformer<List<int>, dynamic> get transformer => new StreamTransformer.fromHandlers(
```

Comment:
Could all these `List<int>` be converted to `UInt8List`? would it be better? Just a thought..